cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/Executor.java

```diff
@@ -918,7 +922,7 @@ public synchronized void executeDemoteProposals(Collection<ExecutionProposal> pr
       initProposalExecution(proposals, demotedBrokers, concurrentSwaps, null, 0,
                             requestedClusterLeadershipMovementConcurrency, requestedBrokerLeadershipMovementConcurrency,
                             requestedExecutionProgressCheckIntervalMs, replicaMovementStrategy, isTriggeredByUserRequest, loadMonitor);
-      startExecution(loadMonitor, demotedBrokers, null, replicationThrottle, isTriggeredByUserRequest);
+      startExecution(loadMonitor, demotedBrokers, null, replicationThrottle, replicationThrottle, isTriggeredByUserRequest);
```

Comment:
What is the implication here to have log dir throttling set to replication throttle here?

Reply:
Mistake; Since it is demote task, we don't need `logDirThrottle` parameter so it should be `null`. fixed.